production/scrapers/lasd.R

```diff
@@ -48,12 +48,29 @@ lasd_pull <- function(x, wait = 5){
 lasd_restruct <- function(x){
     
     # Trim whitespace and brighten image 
-    x <- magick::image_trim(x) %>% 
+    x <- x %>%
+        #magick::image_read() %>%
+        magick::image_trim() %>% 
         magick::image_modulate(brightness = 120)
     
     w_ <- magick::image_info(x)$width
     h_ <- magick::image_info(x)$height
     
+    confirmed.crop <- "570x30+580+400"
+    recover.crop <- '570x30+580+710'
+    death.crop <- '570x30+580+780'
+    quarantine.crop <- "570x30+590+950"
+    drop.neg.asymp.1.crop <- "570x25+600+562"
+    drop.neg.asymp.2.crop <- "570x25+600+585"
+    drop.neg.symp.crop <- "570x25+0+562"
+    drop.pos.asymp.crop <- '570x30+580+502'
+    drop.pos.symp.crop <- "562x25+0+508"
+    drop.test.asymp.1.crop <- '570x30+580+605'
```

Comment:
When I change the height to be 35 here, it works fine. EG `drop.pos.symp.crop <- '570x35+580+605'`

Reply:
Same as above.